doc/man/8/ceph-osd.rst

```diff
@@ -18,14 +18,16 @@ Synopsis
 Description
 ===========
 
-**ceph-osd** is the object storage daemon for the Ceph distributed file
-system. It is responsible for storing objects on a local file system
-and providing access to them over the network.
-
-The datapath argument should be a directory on a xfs file system
-where the object data resides. The journal is optional, and is only
-useful performance-wise when it resides on a different disk than
-datapath with low latency (ideally, an NVRAM device).
+**ceph-osd** is the **o**\bject **s**\torage **d**\aemon for the Ceph
+distributed file system. It is responsible for storing objects on a local file
```

Comment:
It manages data on local storage with redundancy and provides access over the network.

The second paragraph is Filestore legacy, and s/NVMRAM/SSD/

Reply:
Accepted.